src/server/stream_family.cc

```diff
@@ -78,6 +78,8 @@ struct GroupInfo {
   size_t consumer_size;
   size_t pending_size;
   streamID last_id;
+  long long entries_read;
+  long long lag;
```

Comment:
In our (as opposed to Redis') code, we prefer strict types such as `int64_t` over unspecified `long long`

Reply:
fixed